bin/build.ts

```diff
@@ -61,6 +61,10 @@ const argv = yargs
     describe: 'relative path to a tsconfig.json file to compile typescript',
     type: 'string'
   })
+  .option('transpile-only', {
+    describe: 'When using --tsconfig, disable typechecking in ts-loader',
```

Comment:
non-blocking: looks like the other descriptions here start with lower-case
```suggestion
    describe: 'when using --tsconfig, disable typechecking in ts-loader',
```